specification/compute/resource-manager/readme.md

```diff
@@ -106,6 +106,23 @@ directive:
       - $.definitions.VirtualMachineScaleSetUpdate
     suppress:
       - RequiredPropertiesMissingInResourceModel
+  - where:
+      - $.definitions.AvailabilitySetUpdate
+    suppress:
+      - RequiredPropertiesMissingInResourceModel
+  - where:
+      - $.definitions.VirtualMachineExtensionUpdate
+    suppress:
+      - RequiredPropertiesMissingInResourceModel
+  - where:
+      - $.definitions.VirtualMachineUpdate
+    suppress:
+      - RequiredPropertiesMissingInResourceModel
+  - where:
+      - $.definitions.ImageUpdate
+    suppress:
+      - RequiredPropertiesMissingInResourceModel
+

```

Comment:
Context: These models are used as payload for PATCH and not used as response model of an API. This means these models should not have any required properties, hence suppressions are valid AFAIK.  Ref: https://github.com/Azure/azure-rest-api-specs/pull/2428#discussion_r170118869